phpseclib/Crypt/RSA.php

```diff
@@ -1917,7 +1897,6 @@ function _rsassa_pss_sign($m)
      * @access private
      * @param string $m
      * @param string $s
-     * @throws \RuntimeException on invalid signature
      * @return string
```

Comment:
Should this be `@return bool|string` as well? And also at https://github.com/phpseclib/phpseclib/pull/902/files#diff-b4b2acaebb5e3d125481ec5da11baff5R1989 below?